apis/v1alpha1/version.go

```diff
@@ -32,7 +32,8 @@ type VersionSpec struct {
 	CodeSHA256 *string `json:"codeSHA256,omitempty"`
 	// A description for the version to override the description in the function
 	// configuration.
-	Description *string `json:"description,omitempty"`
+	Description               *string                            `json:"description,omitempty"`
+	FunctionEventInvokeConfig *PutFunctionEventInvokeConfigInput `json:"functionEventInvokeConfig,omitempty"`
```

Comment:
Thoughts: how about naming this field `EventInvokeConfig` - how does CF support this?

Reply:
Good suggestion, the CF uses the same naming too `EventInvokeConfig`. Here's [the link](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-lambda-eventinvokeconfig.html)

Reply:
Looks like it's named `FunctionEventInvokeConfig` in https://github.com/aws-controllers-k8s/lambda-controller/blob/main/apis/v1alpha1/function.go#L66

Reply:
To keep the things similar across all resources, I am keeping the naming to be `FunctionEventInvokeConfig`. Will open a PR in future to correct the naming for all resources.